example-cropping/README.md

```diff
@@ -0,0 +1,779 @@
+    const [loading, setLoading] = useState(false);
+    const [error, setError] = useState(false);
+    async function login(e) {
+      e.preventDefault()
+  
+      if (loading) return;
+  
+      setLoading(true)
+      try {
+        setError(false)
+        await props.appwrite.account.createSession(
+          email,
+          password
+        )
+        props.getUserData()
+      } catch(err) {
+        
+        setError(err.message)
+      }
+       setLoading(false)
+    }
+
+  const classes = useStyles();
+
+  return (
+    <Container component="main" maxWidth="xs"  style={{ display: props.currentPage ? "block" : "none" }}>
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Typography component="h1" variant="h5">
+          Login
+        </Typography>
+        
+        {error && (
+             <Alert severity="error">{error}</Alert>
+      )}
+        <form className={classes.form} noValidate onSubmit={(e) => login(e)}>
+          <TextField
+          onChange={(event) => setEmail(event.target.value)} 
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            id="email"
+            label="Email Address"
+            name="email"
+            autoComplete="email"
+            autoFocus
+          />
+          <TextField
+          onChange={(event) => setPassword(event.target.value)}
+            variant="outlined"
+            margin="normal"
+            required
+            fullWidth
+            name="password"
+            label="Password"
+            type="password"
+            id="password"
+            autoComplete="current-password"
+          />
+         
+          <Button
+            type="submit"
+            fullWidth
+            variant="contained"
+            color="primary"
+            className={classes.submit}
+            disabled={loading}
+          >
+            Login
+          </Button>
+          <Grid container>
+            
+            <Grid item>
+              <Link href="#" variant="body2" onClick={()=>props.setCurrentPage(props.currentPage)}>
+                {"Don't have an account? Sign Up"}
+              </Link>
+            </Grid>
+          </Grid>
+        </form>
+      </div>
+    </Container>
+  );
+}
+
+export {Login}
+
+```
+
+example usage
+
+```js
+   <Login
+            appwrite={appwrite}
+            currentPage={currentPage}
+            getUserData={getUserData}
+            setCurrentPage={(currentPage) => setCurrentPage(!currentPage)}
+          />
+```
+it takes four props which is 
+
+- ```appwrite``` to connect to appwrite server using sdk
+- ```currentPage``` the state of login or sign up
+- ```setCurrentPage``` to switch between login and sign up
+- ```getUserData``` to check we already login or not
+
+here we implement login because in order for us to be able to crop an image we should login to our appwrite server and this component does exactly that.
```

Comment:
```suggestion
Here we implement login because in order for us to be able to crop an image we should login to our appwrite server and this component does exactly that.
```